include/rmm/mr/device/per_device_resource.hpp

```diff
@@ -129,20 +156,21 @@ RMM_EXPORT inline auto& get_map()
  * Returns a pointer to the `device_memory_resource` for the specified device. The initial
  * resource is a `cuda_memory_resource`.
  *
- * `id.value()` must be in the range `[0, cudaGetDeviceCount())`, otherwise behavior is undefined.
+ * `device_id.value()` must be in the range `[0, cudaGetDeviceCount())`, otherwise behavior is
+ * undefined.
  *
  * This function is thread-safe with respect to concurrent calls to `set_per_device_resource`,
  * `get_per_device_resource`, `get_current_device_resource`, and `set_current_device_resource`.
  * Concurrent calls to any of these functions will result in a valid state, but the order of
  * execution is undefined.
  *
- * @note The returned `device_memory_resource` should only be used when CUDA device `id` is the
- * current device  (e.g. set using `cudaSetDevice()`). The behavior of a device_memory_resource is
- * undefined if used while the active CUDA device is a different device from the one that was active
- * when the device_memory_resource was created.
+ * @note The returned `device_memory_resource` should only be used when CUDA device `device_id` is
+ * the current device  (e.g. set using `cudaSetDevice()`). The behavior of a
+ * `device_memory_resource` is undefined if used while the active CUDA device is a different device
+ * from the one that was active when the `device_memory_resource` was created.
  *
  * @param device_id The id of the target device
- * @return Pointer to the current `device_memory_resource` for device `id`
+ * @return Pointer to the current `device_memory_resource` for device `device_id`
  */
 inline device_memory_resource* get_per_device_resource(cuda_device_id device_id)
```

Comment:
*question*: I suppose you're not marking any of these functions as `[[deprecated]]` because you want that this PR is _not_ a breaking change?

Reply:
Right. My RMM deprecation approach is typically:
1. Add new APIs
2. Change all of RAPIDS to use the new APIs.
3. Deprecate old APIs.
4. (At least one release later) Remove deprecated APIs.

Reply:
Makes sense, thanks.